protocol/communities/community.go

```diff
@@ -242,6 +245,40 @@ func (o *Community) CreateChat(chatID string, chat *protobuf.CommunityChat) (*Co
 	return changes, nil
 }
 
+func (o *Community) EditChat(chatID string, chat *protobuf.CommunityChat) (*CommunityChanges, error) {
+	o.mutex.Lock()
+	defer o.mutex.Unlock()
+
+	if o.config.PrivateKey == nil {
```

Comment:
I'd suggest to use `!o.IsAdmin()` for better readability

Reply:
Thanks, I didn't realise we had this convenience method. Updated.